README.md

```diff
@@ -33,18 +49,35 @@ npm run dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
-## The Graph data
+## Backend
 
-You can run the following commands to query the Graph Node and retrieve some data. This data will be saved in a JSON file.
+You can run the following commands to query the Envio Node and upload data to neo4j
 
 ```bash
-npm run graph:profile
+npm run graph
+```
+
+https://envio-testing.mainnet.lukso.dev/
+
+This script will fo an initial sweep of the blockchain. It will query all the profiles and all the transactions and add them to the NEO4j database.
+
+Once the initial sweep is complete, the script will retrieve new profiles and transactions every POLLING_INTERVAL_SECONDS.
 
-# or
+### HELPER: Cypher queries
 
-npm run graph:assets
+#### Create relationships
```

Comment:
Super useful! Would be good to put a sample cypher query to retrieve the relationships related to a specific UP too!